mediawiki/stack.yml

```diff
@@ -0,0 +1,21 @@
+# MediaWiki with MariaDB
+#
+# Access via "http://localhost:8080"
+#   (or "http://$(docker-machine ip):8080" if using docker-machine)
+version: '3'
+services:
+  mediawiki:
+    image: mediawiki
+    ports:
+      - 8080:80
+    links:
+      - database
+    volumes:
+      - /var/www/html/images
```

Comment:
Can the web setup just auto create the `LocalSettings.php` file in the correct place when the directory is writable?  It feels a bit clunky to download the file and then have to get it back into the container (which could be on a swarm).

If we are stuck with downloading the file, I'd recommend having a couple lines here similar to this, but it will only work well on a single host.
```yaml
# after initial setup, download LocalSettings.php to the same directory as this yaml and uncomment the following line and use compose to restart the mediawiki service
#      - ./LocalSettings.php:/var/www/html/LocalSettings.php
```

It might also make sense to